From 69c7a43c7613c4f82910da1b54839f6952740449 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 16 Feb 2022 00:20:16 +0800
Subject: [PATCH 31/50] Revert "drm/lima: fix warning when CONFIG_DEBUG_SG=y &
 CONFIG_DMA_API_DEBUG=y"

This reverts commit b207356933f4593071bbd95da2bb12eaa762a66f.
---
 drivers/gpu/drm/lima/lima_device.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/lima/lima_device.c b/drivers/gpu/drm/lima/lima_device.c
index 36c990589427..65fdca366e41 100644
--- a/drivers/gpu/drm/lima/lima_device.c
+++ b/drivers/gpu/drm/lima/lima_device.c
@@ -357,7 +357,6 @@ int lima_device_init(struct lima_device *ldev)
 	int err, i;
 
 	dma_set_coherent_mask(ldev->dev, DMA_BIT_MASK(32));
-	dma_set_max_seg_size(ldev->dev, UINT_MAX);
 
 	err = lima_clk_init(ldev);
 	if (err)
-- 
2.27.0

